app/models/manageiq/providers/hawkular/middleware_manager/event_parser.rb

```diff
@@ -0,0 +1,14 @@
+module ManageIQ::Providers::Hawkular::MiddlewareManager::EventParser
+  def self.event_to_hash(event, ems_id = nil)
+    $mw_log.debug "ems_id: [#{ems_id}] event: [#{event.inspect}]"
+    {
+      :event_type => event[:event_type],
+      :source     => 'HAWKULAR',
+      :timestamp  => event[:timestamp],
+      :message    => event[:message],
+      :ems_ref    => event[:resource], # not sure about this, possible link from event to resource?
```

Comment:
The hash produced here will be changing in the pending PRs that are queued behind this one. If you don't mind just ignore this for now, it doesn't hurt anything and it will soon change.